armi/operators/operator.py

```diff
@@ -65,16 +73,28 @@ class Operator:  # pylint: disable=too-many-public-methods
     cs : CaseSettings object
             Global settings that define the run.
 
+    cycleNames : list
+        The name of each cycle. Cycles without a name are `None`.
+
+    stepLengths : list
```

Comment:
`list of list of float` (see https://numpydoc.readthedocs.io/en/latest/format.html#parameters)

Reply:
Done